Tests/test_fslViewer.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+import unittest
+import viewer
+import sys
+import os
+import shutil
+import glob
+
+class fsl_con_f(unittest.TestCase): #Class for fsl_con_f tests
+	
+	def setUp(self): #Open necessary file
+		self.myString = ""
+		self.data = "data\\fsl_con_f.nidm.ttlTestResults\\postStats.html"
```

Comment:
No problem, changing it now.

________________________________
From: Camille Maumet <notifications@github.com>
Sent: 12 September 2016 16:35:27
To: pwilliams1/nidmresults-fslhtml
Cc: Williams, Peter; Mention
Subject: Re: [pwilliams1/nidmresults-fslhtml] Unit tests (#10)


In Tests/test_fslViewer.py<https://github.com/pwilliams1/nidmresults-fslhtml/pull/10#discussion_r78396333>:

> @@ -0,0 +1,314 @@
> +#!/usr/bin/env python3
> +import unittest
> +import viewer
> +import sys
> +import os
> +import shutil
> +import glob
> +
> +class fsl_con_f(unittest.TestCase): #Class for fsl_con_f tests
> +
> +     def setUp(self): #Open necessary file
> +             self.myString = ""
> +             self.data = "data\\fsl_con_f.nidm.ttlTestResults\\postStats.html"


@pwilliams1<https://github.com/pwilliams1>: could you use os.path.join as described here<https://docs.python.org/2/library/os.path.html#os.path.join> rather than backslashes? This will ensure that the code works with different OS (Windows, Unix...).

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/pwilliams1/nidmresults-fslhtml/pull/10/files/6425bb3a88018d07df0c7bf8d20f7eacfcdbb5ee#r78396333>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATdQEfLGqyh1eoILmGxthnUhDGBI_i6zks5qpXE_gaJpZM4J3_sb>.